docs/synapse/userguides/storm_ref_automation.rstorm

```diff
@@ -359,6 +359,12 @@ is a dictionary which contains the following keys:
         ``$auto.opts.valu``
             The value of the triggering node.
 
+        ``$auto.opts.edge``
```

Comment:
We generally refer to this as "verb". Lets have a quick goback w/analysts about it :+1:

Reply:
I went back and forth on this. `edge` felt more explicit that it now supports light edges, but verb is the right terminology. A goback sounds good :+1: